### DISCUSSION TITLE
Light Unit 2 BOM

### DISCUSSION THREAD

**Discussion by HostoPhoto** [srozum/film_camera_tester#15] [General]
Hi :)

The **light unit 2 (Rev. 4)** BOM in the wiki gives  : 
- Resistor 3.9K 0.125W
- Resistor 4.7K 0.125W

But the **BOM (Rev. 3)** for the light unit 2 in the asset/pcb folder and the schematics pictures give **2x Resistor 4.7K for R3, R4**
Can you clarify wich one it R3 and R4 ?

Thanks a lot :)

**Comment by srozum** [comment]
Both resistors 4.7K. If you look at schematics, what they used for, either would work.
